src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.cpp

```diff
@@ -527,11 +570,15 @@ real64 AcousticWaveEquationSEM::explicitStep( real64 const & time_n,
     rhs[a] = 0.0;
   } );
 
-  if( this->m_outputSeismoTrace == 1 )
+  real64 checkSismo = m_dtSeismoTrace*m_indexSeismoTrace;
+  real64 epsilonLoc = 1e-12;
```

Comment:
```suggestion
  real64 checkSiesmo = m_dtSeismoTrace*m_indexSeismoTrace;
  real64 const epsilonLoc = 1e-12;
```
If you want, you can also define this constant as a `constexpr` at the top of the solver class to make sure it is not buried inside the code.